simplified-tests/src/test/java/org/nypl/simplified/tests/pdf/PdfViewerProviderTest.kt

```diff
@@ -15,30 +15,30 @@ class PdfViewerProviderTest {
   fun supportsPdfBooksWhenEnabled() {
```

Comment:
This is now asserting the opposite, so the name should change to `doesNotSupportPdfBooksWhenDisabled`.